### ISSUE TITLE
dangling shell from coqide

### ISSUE THREAD

**Issue by mprevot** [coq/coq#18034]
#### Description of the problem

Since 8.16.1 (updated from 8.13), coqide on windows opens an empty shell.
Before, only the GUI was opening, no dangling shell was showing.

It's messy.

#### Coq Version

8.16.1, windows

**Comment by Zimmi48** [comment]
How did you install Coq on Windows? With the Coq Platform binary installer? And what version of Windows are you using? @MSoegtropIMC are you aware of this behavior?

**Comment by MSoegtropIMC** [comment]
Yes this is so. I believe this came with the removal of the wrapper script which existed once and was no longer required after Enrico implemented the environment.txt file support. The wrapper gave a bit more control over this.

Several GTK apps I use on Windows have the same effect. I guess what is happening under the hood is that GTK applications are technically console applications (with main) rather than Windows applications (with WinMain).
 
The shell is btw. closed when CoqIDE closes.

**Comment by mprevot** [comment]
I updated th OP with the demanded details.

If you have a window with something drawn inside, there must be an instance of a win32 window (https://learn.microsoft.com/en-us/windows/win32/learnwin32/creating-a-window)

wether it's coming from shell or winmain; one can create one for instance from a c# program and inject the win32 window in a WPF Rectangle or something else.

There should be a way to make it clean.

**Comment by Zimmi48** [comment]
I'm moving the issue to the Coq Platform repository, since this issue seems specific to the Platform Windows installer.
